docs/CONTRIBUTING.md

```diff
@@ -55,6 +56,59 @@ This describes the _minimum_ behavior expected from all contributors.
 
 [coc]: https://www.rust-lang.org/policies/code-of-conduct
 
+
+Within the repositories, you'll find the following directories and files,
+logically grouping common assets and providing both compiled and minified
+variations.
```

Comment:
Why is this under the code of conduct? I'm guessing this was supposed to be under the `Repositories` section?